tests/test_integration.py

```diff
@@ -940,27 +940,5 @@ def test_contained_within_context(self):
         )
 
 
-class UsNationTestCase(unittest.TestCase):
```

Comment:
The only change here is I moved this test to `test_geo_integration.py`, which is the proper place for it.